ui/js/models/job.js

```diff
@@ -30,8 +30,9 @@ treeherder.factory('ThJobModel', [
             config = config || {};
             var timeout = config.timeout || null;
             var fetch_all = config.fetch_all || false;
+            var uri = config.uri || ThJobModel.get_uri(repoName);
```

Comment:
Could you add a comment explaining why we sometimes use `config.uri` and something `get_uri()` ?